src/reviewer/reviewer.spec.ts

```diff
@@ -352,10 +363,24 @@ describe('Should test identifyReviewers: ', () => {
       createdBy: 'unknown',
       rulesByCreator,
       fileChangesGroups: ['file-group-common'],
+      absentReviewersLogins: [],
     });
     expect(result).to.be.deep.equal([], 'Array should be empty');
     done();
   });
+  it('Should not assign absent reviewers', (done) => {
```

Comment:
maybe this is the case when we have a conflict between already assigned (requestedReviewerLogins) users & absentReviewersLogins